common/src/main/java/com/wynntils/models/experience/CombatXpModel.java

```diff
@@ -38,10 +40,19 @@ public class CombatXpModel extends Model {
 
     private boolean firstJoinHappened = false;
 
+    private final TimedSet<Float> lastMinuteXpGainRawSet = new TimedSet<>(1, TimeUnit.MINUTES, true);
```

Comment:
I understand what you mean, but "last minute" sounds like it is almost too late, [cf](https://www.dictionary.com/browse/at-the-last-minute).

Maybe "latestMinute"? Maybe we need assistance from a native English speaker. @P0keDev ?

Reply:
@magicus I renamed it. Do you accept it so I can merge this (and come back to it if needed)?